_posts/2020-12-16-Hackathon-4-Success.md

```diff
@@ -3,6 +3,6 @@ Our latest Hackathon was as a huge success!  We added many new features to the M
 After the hackathon, we hosted our first guest speaker, Miles Tiglao (USMC, ret.)!  Checkout the recording of his talk at the links below:
 
 > For our non-CAC members, [Zoom Recording](https://us02web.zoom.us/rec/share/8qntZYHsw4H1xkvCFS6YfwuYNu9eweyhi22OUYHwsQJwbFywtCqadBkYzQ_0nsFY.Szee4QhLQhLRYIGP)
-* Passcocde: Z?E$C28k
+* Passcode: Z?E$C28k
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)

Reply:
Good bot. These are meant to be hard coded, information is public and meant to distribute Zoom recording.